From frank.hess@nist.gov  Thu Feb 25 15:56:30 2010
From: Frank Mori Hess <frank.hess@nist.gov>
Date: Mon, 22 Feb 2010 15:32:32 -0500
Subject: Staging: comedi: Increased timeout for digital output dma loading
To: Greg KH <gregkh@suse.de>
Cc: abbotti@mev.co.uk
Message-ID: <20100222203232.GA27165@tailpipe.nist.gov>

Increased timeout for digital output dma loading, due to report of it
being too short for some hardware.

Signed-off-by: Frank Mori Hess <fmhess@users.sourceforge.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/comedi/drivers/ni_mio_common.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/staging/comedi/drivers/ni_mio_common.c
+++ b/drivers/staging/comedi/drivers/ni_mio_common.c
@@ -3795,7 +3795,7 @@ static int ni_cdo_inttrig(struct comedi_
 #endif
 	int retval = 0;
 	unsigned i;
-	const unsigned timeout = 100;
+	const unsigned timeout = 1000;
 
 	s->async->inttrig = NULL;
 
